src/main/java/net/javadiscord/javabot/systems/qotw/model/QOTWSubmission.java

```diff
@@ -0,0 +1,36 @@
+package net.javadiscord.javabot.systems.qotw.model;
+
+import lombok.Data;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+
+import java.util.function.Consumer;
+
+/**
+ * Simple data class that represents a single {@link ThreadChannel submission} and the corresponding
+ * {@link User author}.
+ */
+@Data
```

Comment:
Maybe use a record?

Reply:
records dont allow instance fields tho, ig ill just leave it like that

Reply:
What are they needed for in this case?